scripts/bootstrap_snp.sh

```
@@ -0,0 +1,187 @@
+#!/bin/bash
+set -e
+
+source_common() {
+    local script_dir=$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)